mailer.js

```diff
@@ -162,7 +169,8 @@ module.exports = function (log) {
         email: message.email,
         link: link,
         supportUrl: this.supportUrl,
-        supportLinkAttributes: this._supportLinkAttributes()
+        supportLinkAttributes: this._supportLinkAttributes(),
+        passwordChangeUrl: this._initiatePasswordChange()
```

Comment:
naming nit: `passwordChangeLinkAttributes` or similar for consistency, since it's not just the URL?